README.md

```diff
@@ -93,5 +98,476 @@ vault kv get -field='username' secret/postgres
 vault kv get -field='password' secret/postgres
 ```
 
```

Comment:
@dangernil I think the whole README.md file is wrong. It supposed to describe postgres module, not template